src/app/desktop/components/Instances/index.js

```diff
@@ -23,9 +23,42 @@ const SubNoInstance = styled.div`
   font-size: 15px;
   margin-top: 20px;
 `;
+const sortAlphabetical = instances =>
+  instances.sort((a, b) => (a.name > b.name ? 1 : -1));
+
+const sortByLastPlayed = instances =>
+  instances.sort((a, b) => (a.lastPlayed < b.lastPlayed ? 1 : -1));
+
+const sortByMostPlayed = instances =>
+  instances.sort((a, b) => (a.timePlayed < b.timePlayed ? 1 : -1));
+
+const getInstances = () => {
+  // Data normalization for missing fields
+  const instances = useSelector(_getInstances).map(instance => {
```

Comment:
This will not work because `useSelector` doesn't have access to the state

Reply:
Good catch - not sure why it would still be working since logically it shouldn't, but I just cleaned this up in 7c806f4